src/ExportableMaterial.h

```diff
@@ -88,6 +90,12 @@ class ExportableMaterialPBR : public ExportableMaterialBasePBR {
 
     template <class MFnShader>
     void convert(ExportableResources &resources, const MObject &shaderObject);
+    void loadAiStandard(ExportableResources &resources,
+                        const MFnDependencyNode &shaderNode);
+
+    MStatus tryCreateRoughessMetalnessTexture(ExportableResources &resources, const ExportableTexture &metallicTexture,
```

Comment:
Move to be with the try create normal texture?

Reply:
> Move to be with the try create normal texture?

Not sure what you mean with this.

The code looks very good, only a little typo in 
`tryCreateRoughessMetalnessTexture`, should be 
`tryCreateRoughnessMetalnessTexture`